src/entities/trades/0x/0xTrade.ts

```diff
@@ -123,7 +124,7 @@ export class ZeroXTrade extends TradeWithSwapTransaction {
         }&slippagePercentage=${new Percent(
           maximumSlippage.numerator,
           JSBI.multiply(maximumSlippage.denominator, JSBI.BigInt(100))
-        ).toFixed(4)}`
+        ).toFixed(4)}&feeRecipient=${REFFERER_ADDRESS_CHAIN_MAPPING[chainId]}&buyTokenPercentageFee=${fee}`
```

Comment:
Should we had a fallback in case `REFFERER_ADDRESS_CHAIN_MAPPING` returns `undefined`?